waiter/src/waiter/settings.clj

```diff
@@ -45,6 +45,7 @@
                                           (s/required-key :failed-check-threshold) schema/positive-int}
    (s/required-key :host) schema/non-empty-string
    (s/required-key :hostname) schema/non-empty-string
+   (s/optional-key :alternate-hostnames) [schema/non-empty-string]
```

Comment:
Lexicographic order :(

Reply:
I think this should be next to hostname, and after it.

Reply:
But if it is going to hang up the review, I will just sort it.